pid_controller/src/pid_controller.cpp

```diff
@@ -96,6 +96,8 @@ void PidController::update_parameters()
     return;
   }
   params_ = param_listener_->get_params();
+
+  feedforward_mode_enabled_.writeFromNonRT(params_.enable_feedforward);
```

Comment:
This one is missing when the feedforward is modified online, so we will have to add writeFromRT to update method. What do you think?

Reply:
Sorry all good, I see that's what it is doing